apps/pano/app/features/oauth/OAuthLoginForm.tsx

```diff
@@ -15,6 +15,7 @@ export const OAuthLoginForm: FC<Props> = ({ provider }) => {
           display: "flex",
           alignContent: "center",
           gap: "0.5rem",
+          "@media (max-width: 400px)": { width: "100%" },
```

Comment:
These shouldn't be here, we should define the breakpoints in the kampus-ui package's theme object, then we should use those breakpoints here, not a random breakpoint inside a component. Let's fix that.